lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -652,64 +650,16 @@ class PatternStopContents extends Component<Props, State> {
             <FormGroup bsSize='small'>
               <ControlLabel className='small'>{patternStopCardMessages('PatternStopContents.defaultTimeInLocation')}</ControlLabel>
               <MinuteSecondInput
-                seconds={this.state.flexDefaultZoneTime}
+                seconds={this.state.defaultDwellTime}
                 onChange={(newValue) => this._onMinuteSecondInputChange(
                   newValue,
-                  'flexDefaultZoneTime'
+                  'defaultDwellTime'
```

Comment:
Not sure where it happens in the code, but there is a layout glitch with dwell time:

<img width="237" alt="image" src="https://github.com/user-attachments/assets/39a93298-7930-4651-8bc7-1c3d7488db2e">